### ISSUE TITLE
about code

### ISSUE THREAD

**Issue by jiangyanjavawin** [ace19-dev/image-retrieval-tf#4]
Hello sir,I am very excited to find your code,it's really beautiful,I have a question want to ask you,I find you add the SE in the resnet block,How to add it at the end of the model,I think likely fine tune.Thanks your share again,you are a nice guy.

**Comment by ace19-dev** [comment]
hi. jiangyanjavawin

'How to add it at the end of the model' 
i don't understand what you said.

**Comment by jiangyanjavawin** [comment]
Hello,thanks for your reply,my mean is add the SE at the end of the last bottleneck instead of like you did inner the&nbsp; bottleneck.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Sean Kim"<notifications@github.com&gt;;
发送时间:&nbsp;2020年4月18日(星期六) 晚上8:17
收件人:&nbsp;"ace19-dev/image-retrieval-tf"<image-retrieval-tf@noreply.github.com&gt;;
抄送:&nbsp;"飞天小魔杖"<1164403787@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [ace19-dev/image-retrieval-tf] about code (#4)





 
hi. jiangyanjavawin
 
'How to add it at the end of the model'
 i don't understand what you said.

**Comment by ace19-dev** [comment]
Hi. jy

if you want to use SE module in last bottleneck, you can make another SE bottleneck function then use it in resnet model. 

In conclusion, you need to modify resnet_v2.py.

thank you.

**Comment by jiangyanjavawin** [comment]
Thank you very much,I will try it later.



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Sean Kim"<notifications@github.com&gt;; 
发送时间: 2020年4月18日(星期六) 晚上8:45
收件人: "ace19-dev/image-retrieval-tf"<image-retrieval-tf@noreply.github.com&gt;; 
抄送: "飞天小魔杖"<1164403787@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [ace19-dev/image-retrieval-tf] about code (#4)





 
Hi. jy
 
if you want to use SE module in last bottleneck, you can make another SE bottleneck function then use it in resnet model.
 
In conclusion, you need to modify resnet_v2.py.
 
thank you.
